src/AppBundle/Service/EventProcessor.php

```diff
@@ -246,11 +252,13 @@ private function setContributionStats(): void
             $this->createOrUpdateEventStat('pages-created', $this->pagesCreated);
             $this->createOrUpdateEventStat('pages-improved', $this->pagesImproved);
             $this->createOrUpdateEventStat('byte-difference', $this->byteDifference);
+            $this->createOrUpdateEventStat('pages-created-pageviews', $this->pagesCreatedPageviews);
```

Comment:
This is not really contribution stats.

Reply:
What do you mean?

Reply:
It should be in a function other than setContributionStats()